src/features/profile/skills/add-skills/form-content.tsx

```diff
@@ -0,0 +1,37 @@
+import MenuItem from "@mui/material/MenuItem";
+import Button from "@mui/material/Button";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import useAllSkills from "./use-all-skills";
+import { useAuthState } from "hooks/use-auth-state";
+import useForm from "./use-form";
+import { useState } from "react";
+
+const FormContent = ({ setIsOpen }: AddSkillFormContentProps) => {
+  const [selectedSkillId, setSelectedSkillId] = useState("");
+  const { token } = useAuthState();
+  const { allSkills } = useAllSkills(token!);
```

Comment:
Creeria puedes dejar el hook de obtencion del token dentro, me parece mas responsabilidad de ese mismo hook mas que del componente en si
```suggestion
  const allSkills = useAllSkills();
```